.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +1,33 @@
 ---
-name: Bug report
+name: Bug Report
```

Comment:
can this change actually be made? Im not sure if this title is case sensitive as the format is specified by github

Reply:
I'll try to check somewhere, I just went in, spotted a non-capitalised title style naming, and went for it.

Reply:
This seems to have Bug Report capitalised in the example, so I'd say it should fly? https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository